src/main/scala/com/gu/salesfoce/messageHandler/Config.scala

```diff
@@ -0,0 +1,47 @@
+package com.gu.salesfoce.messageHandler
+
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.auth.{ AWSCredentialsProviderChain, DefaultAWSCredentialsProviderChain }
+import com.amazonaws.services.s3.AmazonS3ClientBuilder
+import com.typesafe.config.ConfigFactory
+
+import scala.io.Source
+
+object Config {
+  case class Env(app: String, stack: String, stage: String) {
+    override def toString: String = s"App: $app, Stack: $stack, Stage: $stage\n"
+  }
+
+  object Env {
+    def apply(): Env = Env(
+      Option(System.getenv("App")).getOrElse("DEV"),
+      Option(System.getenv("Stack")).getOrElse("DEV"),
+      Option(System.getenv("Stage")).getOrElse("CODE"))
+  }
+
+  val stage = Env().stage.toUpperCase
+
+  val credentialsProvider = new AWSCredentialsProviderChain(
+    new ProfileCredentialsProvider("membership"),
+    new DefaultAWSCredentialsProviderChain())
+
+  val s3Client = AmazonS3ClientBuilder
+    .standard()
+    .withCredentials(credentialsProvider)
+    .build()
+
+  val configData = {
```

Comment:
in my lambda, I'm just using them as parameters to the cloudformation (e.g. presto url, mailto address), but soon it will get to the point where I have to steal this idea too.